src/openforms/config/models.py

```diff
@@ -191,6 +192,15 @@ class GlobalConfiguration(SingletonModel):
             "'(optional)' and required fields are unmarked."
         ),
     )
+    form_upload_default_file_types = ArrayField(
+        models.CharField(max_length=12, choices=mimetypes.types_map.items()),
+        verbose_name=_("Default allowed file upload types"),
+        help_text=_(
+            "Provide a list of default allowed file upload types. If empty, all extensions are allowed."
```

Comment:
> If empty, all extensions are allowed.

Is this true? I thought you need to pass `*` (All) as an option in the extensions list.

Reply:
Yes, this is correct. `*` and an empty list are equivalent.